components/PageHeader.jsx

```diff
@@ -83,64 +33,30 @@ const PageHeader = ({
 
     const stickyClass = isSticky ? ' usda-page-header--sticky' : '';
 
-    const showToolbar = (
-        !isEmpty(fyProps) ||
-        !isEmpty(shareProps) ||
-        !isEmpty(downloadProps)
-    );
-
     return (
-        <main id={id} className={`usda-page__container${classNames ? ` ${classNames}` : ''}`}>
-            <section className={`usda-page-header${stickyClass}`} ref={stickyHeader}>
-                <div className="usda-page-header__container">
-                    <div className="usda-page-header__header">
-                        {overLine && <strong className="usda-page-header__overline">{overLine}</strong>}
-                        <div className="usda-page-header__title">
-                            <h1>{title}</h1>
-                        </div>
+        <section className={`usda-page-header${stickyClass}`} ref={stickyHeader}>
```

Comment:
Why is this section tag instead of header?

Reply:
We already have a header:

![image](https://user-images.githubusercontent.com/12897813/115926545-03eb7180-a451-11eb-8ef2-8dec71711d01.png)

Reply:
True, for the page template. This looks like a header for the "actual" page as well.